front/src/components/header/subcomponents/ToggleColorModeButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { IconButton, useColorMode } from '@chakra-ui/core';
+
+const ToggleColorModeButton = () => {
+  const { colorMode, toggleColorMode } = useColorMode();
+
+  return (
+    <IconButton
+      aria-label="toggle color mode"
+      ml=" 12px"
+      onClick={toggleColorMode}
+      icon={colorMode === 'light' ? 'moon' : 'sun'}
```

Comment:
nit - alphabetize props names

Reply:
done https://github.com/trybick/tv-minder/pull/32/commits/b1914d8d6ea08ac6a748bdd8726afb966e056563

Reply:
@trybick if you want I can add https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-sort-props.md to enforce it everywhere 😄

Reply:
Thank you. Yes please let's enforce it everywhere 🚀 . that would be great!

Reply:
@luisFilipePT Did you still want to make a PR for this eslint rule?

Reply:
Yes, I can do it this afternoon 😄

Reply:
@trybick btw I am also looking at the color function and one question, does the color needs to be static (defined at the beginning) or can it be generated? as long as it is always the same color for the same id?

Reply:
great! 

for the random color for the shows on the calendar? Hmm, it generates the color based on the ID, so I think every time the same ID will have the same color.  

The `return index + 1` was just a hack to get it to work if there are two IDs with the same index but I would like to get it working so if there are 3 or more shows with the same index they will all have a unique index.